signup.py

```diff
@@ -18,4 +19,4 @@ def signup(client,message):
         #
         # Ask Randium or somebody else if you have any questions.
     else:
-        #Should return a Mailbox, though I'm confused on the implementation
+        return Mailbox().respond("You can't sign up when a game's running, sweetie. :kissing_hearts:",message.channel)
```

Comment:
that `:kissing_hearts:` thing makes me want to barf and needs to go 😛

Reply:
You're right. I looked it up, and it's kissing_heart, not kissing_hearts. 😘